src/content/bqtools-intro/steps/Step6.md

```diff
@@ -0,0 +1,28 @@
+<script>
+import Execute from "$components/Execute.svelte";
+import Link from "$components/Link.svelte";
+</script>
+
+Now we have a few BINSEQ files - what can we do with them?
+
+If we want to explore the underlying sequence data we can make use of the `binseq decode` command.
+This will let us output the sequence data in a human-readable format.
+
+Let's output a few sequences as a tab-delimited file (TSV).
+
+<Execute command="bqtools decode ./fastq/sample1.vbq | head -n 20" />
+
+Notice that the first column is the record ID and the second column is the sequence itself.
+You'll notice that the record ID is repeated every two lines for this file - this is because the R1 and R2 sequence data currently have the same record ID (it's position in the file).
```

Comment:
Should that be "its" instead?

```suggestion
You'll notice that the record ID is repeated every two lines for this file - this is because the R1 and R2 sequence data currently have the same record ID (its position in the file).
```